gravatar/src/test/java/com/gravatar/AvatarUrlTest.kt

```diff
@@ -234,7 +245,19 @@ class AvatarUrlTest {
                 AvatarQueryOptions(
                     defaultAvatarOption = CustomUrl("https://example.com/?encoded=true&please=yes"),
                 ),
-            ).uri().toString(),
+            ).url().toString(),
+        )
+    }
+
+    @Test
+    fun `Force default avatar false must generate f=n`() {
```

Comment:
❓ Should we use lowercase for the first letter in the fun name? (Except in the other test where is the class name `AvatarUrl`)

```suggestion
    fun `force default avatar false must generate f=n`() {
```

Reply:
Fixed in [30a7c5f](https://github.com/Automattic/Gravatar-SDK-Android/pull/82/commits/30a7c5fc3b18440bbd1472e92b74adc875979655)